tests/helpers.py

```diff
@@ -356,7 +354,7 @@ def verify_block_pool_exists(pool_name):
 
 def get_admin_key():
     """
-    Fetches admin key secret from Ceph
+    Fetches admin key secret from CEPH
```

Comment:
Why these docstring changes are needed?

Reply:
Yes these changes were suggested in old pr

Reply:
I believe the suggestion was to capitalize `Ceph` and all caps abbreviations like `OCP`. Ceph doesn't need to be all caps.

Reply:
Done